teams/rust-by-example.toml

```diff
@@ -0,0 +1,13 @@
+name = "rust-by-example"
+subteam-of = "docs"
+
+[people]
+leads = ["steveklabnik"]
+members = [
+    "steveklabnik",
+    "marioidival",
+]
+
+[website]
+name = "Rust by Example team"
+description = "working on the rust by example"
```

Comment:
Probably a better description here would be something like "maintaining and updating Rust By Example".

Reply:
done!